src/js/controllers/chatController.js

```diff
@@ -63,53 +63,51 @@ module.exports = ViewController.extend({
     },
 
     sendMessage: function(text) {
-        var conversationDeferred = $.Deferred();
-        var messageDeferred = $.Deferred();
+        return $.Deferred().resolve().then(function() {
+            var promise = $.Deferred();
 
-        if (this.conversation.isNew()) {
```

Comment:
_initFaye and createConversation seemed to be execute at the same thing, which caused faye to be subscribed too late